app/main.py

```diff
@@ -2,0 +1,54 @@
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    FrontendDeveloper_skills = ["JavaScript", "HTML", "CSS"]
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        for skill in self.FrontendDeveloper_skills:
+            self.skills.append(skill)
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    BackendDeveloper_skills = ["Python", "SQL", "Django"]
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        for skill in self.BackendDeveloper_skills:
+            self.skills.append(skill)
+
+    def create_powerful_api(self) -> str:
+        print(f"{self.name} is creating an API...")
+        return "http://127.0.0.1:8000"
+
+
+class AndroidDeveloper(SoftwareEngineer):
+    AndroidDeveloper_skills = ["Java", "Android studio"]
```

Comment:
The `AndroidDeveloper_skills` should be accessed as a class attribute using `self.__class__.AndroidDeveloper_skills` in the `__init__` method.